site/docs/overview.md

```diff
@@ -37,13 +37,13 @@ Usage: very_good create <subcommand> <project-name> [arguments]
 -h, --help    Print this usage information.
 
 Available subcommands:
-  dart_cli          Creates a new very good Dart CLI in the specified directory.
-  dart_package      Creates a new very good Dart package in the specified directory.
-  docs_site         Creates a new very good docs site in the specified directory.
-  flame_game        Creates a new very good Flame game in the specified directory.
-  flutter_app       Creates a new very good Flutter app in the specified directory.
-  flutter_package   Creates a new very good Flutter package in the specified directory.
-  flutter_plugin    Creates a new very good federated Flutter plugin in the specified directory.
+  dart_cli          Generate a Very Good Dart CLI application.
+  dart_package      Generate a reusable Dart package.
```

Comment:
I think we should be consistent and either put Very Good in all of them or remove it from all. Right now some are not very good (dart_package, flutter_package, and flutter_plugin) but it's unclear why.